CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Feature: Add new `TextInputWithLimit` React component.
 - Fix: Bring back `@include k-Tooltip;` in _components.scss.
```

Comment:
Le texte est en _italique_ dans la relecture de PR, ça doit être dû à la ligne du dessus. Tu peux en profiter pour remplacer `in _components.scss.` par ```in `_components.scss`.``` ?